config/webpack/terra-dev-site.config.js

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Could we rename to terra-dev-site.webpack.config.js?

Reply:
sure